include/mbedtls/config.h

```diff
@@ -2254,6 +2254,19 @@
  */
 #define MBEDTLS_ECP_C
 
+/**
+ * \def MBEDTLS_USE_UECC
+ *
+ * Enable the tinycrypt ECC library.
+ *
+ * Module:  tinycrypt/ecc.c
+ *          tinycrypt/ecc_dh.c
+ *          tinycrypt/ecc_dsa.c
+ *
```

Comment:
We should mention here how this option relates Mbed TLS' native ECP modules.

Reply:
Done